ethereum/p2p/src/main/java/org/hyperledger/besu/ethereum/p2p/discovery/internal/PeerDiscoveryController.java

```diff
@@ -447,6 +456,7 @@ private boolean addToPeerTable(final DiscoveryPeer peer) {
       peerTable.tryAdd(peer);
     }
 
+    numPeersBonded.getAndIncrement();
```

Comment:
maybe we should check the AddResult again to increment the metric?

Reply:
removed